content/tutorial/godot/2D/normal-map-hints/index.md

```diff
@@ -0,0 +1,224 @@
+
++++
+title = "Normal Map Hints"
+description = "Lear how to use Normal Maps with Godot for 2D games."
+author = "Pablo"
+
+date = 2020-09-18T18:12:50-03:00
+weight = 5
+draft = true
+
+difficulty = "beginner"
+keywords = ["tutorial", "2D"]
++++
+
+## What will you learn
+
+*Normal maps* are textures used to increase the detail of dynamic lighting on surface.
+With them, you can make a flat surface look like a complex geometry, with relief and
+texture, when interacting with lights.
+
+In this tutorial you will learn some common use cases of *normal maps* in Godot engin
+for 2D games, and I will tell you some considerations you should know to make the
+most of this powerful technique.
+
+You can get the sprites to reproduce the following examples [here](https://github.com/GDQuest/godot-mini-tuts-demos/releases/download/untagged-1f11c44a177b76d8c6b0/NormalMapHintsAssets.zip).
+
+## General Concepts
+
+A *normal map* is a 2D texture in which each pixel color represents a *normal vector*.
```

Comment:
This first line is already enough for people who want to learn about lighting 2d characters in Godot.

The idea with these tuts is to go straight to the meat of the info, and link to pages that can provide more technical or theoretical information.